src/wizard/mod.rs

```diff
@@ -581,8 +581,15 @@ pub async fn run_wizzard_run(mut config: Settings) -> Result<(), String> {
             return Err(err.to_string());
         }
     }
+    // TODO: stop using useless param
     let ide_conf_path = ide_conf_path_tmp.join("esp_ide.json");
-    config.save_esp_ide_json(ide_conf_path.to_str().unwrap())?;
+    match config.save_esp_ide_json(ide_conf_path.to_str().unwrap()) {
```

Comment:
Probably make sense to rename this function also.

Reply:
this is a key from the library so it can not be changed here. Also refactoring of library, CLI and GUI for the purpose of renaming single variable does not make sense.